website/docs/tutorials/08-interacting-with-starknet/01-invoke.md

```diff
@@ -34,7 +34,7 @@ Setting `max-fee` to `auto` is discouraged, since it may incur extra unexpected
 :::
 
 :::note
-Inputs have to be passed as list of felts (integers), like Cairo calldata.
+Inputs have to be passed either as a list of felts (integers), like Cairo calldata, or as a dict with arguments' names mapped to their values.
```

Comment:
Wouldn't be useful top have an example here?

Reply:
[address feedback for docs](https://github.com/software-mansion/protostar/pull/1140/commits/259a8d00813769fe9016075045ddc1a16473b53c)